chapters/ch45.md

```diff
@@ -1,38 +1,35 @@
 
---> _replace THIS LINE by your translation for the above line_
+Giả sử y* là đầu ra "chính xác" nhưng thuật toán thay vào đó đưa ra đầu ra y<sub>out</sub>. Sau đó, bài kiểm tra chính để đo xem liệu Điểm<sub>x</sub>(y*) > Điểm<sub>x</sub>(y<sub>out</sub>) không. Nếu bất đẳng thức này đúng, thì chúng ta đổ lỗi cho thuật toán tối ưu hóa cho kết quả sai. Tham khảo chương trước để đảm bảo bạn hiểu logic đằng sau điều này. Ngược lại, chúng ta đổ lỗi cho tính toán của Điểm<sub>x</sub>(y).
 
-> Let’s look at one more example. Suppose you are building a Chinese-to-English machine
-> translation system. Your system works by inputting a Chinese sentence ​*C*,​ and computing
-> some Score​<sub>C</sub>​(​*E*)​ for each possible translation ​E.​ For example, you might use Score​<sub>C</sub>​(​*E*)​ = P(*E*|*C*), the probability of the translation being E given that the input sentence was ​*C*.
+> Let’s look at one more example. Suppose you are building a Chinese-to-English machine translation system. Your system works by inputting a Chinese sentence ​*C*,​ and computing some Score​<sub>C</sub>​(​*E*)​ for each possible translation ​E.​ For example, you might use Score​<sub>C</sub>​(​*E*)​ = P(*E*|*C*), the probability of the translation being E given that the input sentence was ​*C*.
 
---> _replace THIS LINE by your translation for the above line_
+Hãy để xem xét một ví dụ nữa. Giả sử bạn đang xây dựng một hệ thống dịch máy từ tiếng Trung sang tiếng Anh. Hệ thống của bạn hoạt động bằng cách nhập một câu tiếng Trung *C* và tính toán một số Điểm<sub>C</sub>(*E*) cho mỗi bản dịch có thể E. Ví dụ: bạn có thể sử dụng Điểm<sub>C</sub>(*E*) = P(*E*|*C*), xác suất dịch được E cho rằng câu đầu vào là *C*.
 
 > Your algorithm translates sentences by trying to compute:
 
---> _replace THIS LINE by your translation for the above line_
+Thuật toán của bạn dịch các câu bằng cách cố gắng tính toán:
 
 ![img](../imgs/C45_01.png)
 
 > However, the set of all possible English sentences ​*E* i​ s too large, so you rely on a heuristic search algorithm.
 
---> _replace THIS LINE by your translation for the above line_
+Tuy nhiên, tập hợp các câu tiếng Anh có thể *E* quá lớn nên bạn dựa vào thuật toán tìm kiếm thực nghiệm.
 
 > Suppose your algorithm outputs an incorrect translation ​*E​*<sub>out</sub>​ rather than some correct translation ​E​*. Then the Optimization Verification test would ask you to compute whether Score​<sub>C</sub>​(*E**) > Score​<sub>C</sub>​(*E*<sub>out</sub>). If this inequality holds, then the Score​<sub>C</sub>​(.) correctly recognized E* as a superior output to *E​*<sub>out</sub>​; thus, you would attribute this error to the approximate search algorithm. Otherwise, you attribute this error to the computation of Score​<sub>C</sub>​(.).
```

Comment:
Câu thứ hai bị in nghiêng lỗi do dấu `*` ở cuối câu thứ nhất (đây là do quy ước của markdown, đoạn văn giữa hai dấu `*` sẽ bị in nghiêng). Để sửa việc này, bạn thay `E*` bằng `E\*` để nói với markdown rằng mình muốn dấu `*` thực sự chứ không phải dấu in nghiêng. Bạn xem bản gốc và sửa những chỗ tương ứng nhé.